src/coreclr/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -13,7 +13,7 @@
     <_ILLinkTrimXmlFilePath Condition=" '$(_ILLinkTrimXmlFilePath)' == '' ">$(MSBuildThisFileDirectory)ILLinkTrim.xml</_ILLinkTrimXmlFilePath>
     <_ILLinkTasksToolsDir>$(PkgILLink_Tasks)/tools</_ILLinkTasksToolsDir>
     <_ILLinkTasksDir>$(_ILLinkTasksToolsDir)/$(NetFrameworkCurrent)/</_ILLinkTasksDir>
-    <_ILLinkTasksDir Condition="'$(MSBuildRuntimeType)' == 'Core'">$(_ILLinkTasksToolsDir)/netcoreapp2.0/</_ILLinkTasksDir>
+    <_ILLinkTasksDir Condition="'$(MSBuildRuntimeType)' == 'Core'">$(_ILLinkTasksToolsDir)/netcoreapp3.0/</_ILLinkTasksDir>
```

Comment:
We should instead remove the property that calculates the path to the illink directory and instead use the defined one that comes via the package: https://github.com/mono/linker/blob/e5d6b69677dfc0da19c5b2c96b3466e8d5016327/src/ILLink.Tasks/ILLink.Tasks.targets#L8

Reply:
Good suggestion but as this needs more work it should be done in different PR (tracking issue #32453)

Reply:
works for me, thanks for opening the issue.